### DISCUSSION TITLE
access permissions for table joins

### DISCUSSION THREAD

**Discussion by kalmarz** [prest/prest#508] [API Server] [answered]
I have 2 tables in the db and `prestd` is configured to be able to access ALL fields:

[access]
restrict = true

[[access.tables]]
name = "scans"
permissions = ["read", "write", "delete"]
fields = ["scancode", ...]

[[access.tables]]
name = "tags"
permissions = ["read", "write", "delete"]
fields = ["tag", "scancode", ...]

Yet, the query below dies with:
```yaml
{
    "error": "you don't have permission for this action, please check the permitted fields for this table"
}
```

query: `scans?_select=scans.scancode,tags.tag&_join=inner:tags:scans.scancode:$eq:tags.scancode`

Any idea what I might miss?

**Comment by avelino** [comment, marked as answer]
great point to write a test, I don't know how it will behave, but we probably have a permission problem as you reported
